oap-server/server-receiver-plugin/otel-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/otel/oc/OCMetricHandler.java

```diff
@@ -107,6 +111,15 @@
         return result;
     }
 
+    private static Map<String, String> buildLabelsWithNodeInfo(Node node,
```

Comment:
Please don't hide `buildLabels` in the `buildLabelsWithNodeInfo`, it is misleading the reader. Do like this `buildLabelsFromNodeInfo(buildLables(...))`.